apps/sms/views/inbox/js/inbox.js

```diff
@@ -55,7 +46,7 @@ var InboxView = {
       'check-uncheck-all-button','composer-link',
       'delete-button', 'edit-header','options-button',
       'settings-button','edit-mode', 'edit-form',
-      'draft-saved-banner'
+      'draft-saved-banner', 'list'
```

Comment:
I see, I was thinking that list items are rendered inside shadow DOM and couldn't/shouldn't be exposed to the consumer.... Looks like I just don't understand how it works :)